packages/invoices/src/OutboundShipment/DetailView/DetailViewToolbar.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { FC } from 'react';
+import {
+  AppBarContentPortal,
+  Box,
+  InputWithLabelRow,
+  BasicTextInput,
+  TabList,
+  Tab,
+  useTranslation,
+} from '@openmsupply-client/common';
+import { CustomerSearchInput } from '../CustomerSearchInput';
+import { OutboundShipment } from './types';
+
+interface OutboundShipmentToolbarProps {
+  draft: OutboundShipment;
+  currentTab: string;
+  onChangeTab: (newTab: string) => void;
+}
+
+export const OutboundShipmentToolbar: FC<OutboundShipmentToolbarProps> = ({
```

Comment:
The name of the component and the filename don't align - and I think `OutboundShipmentToolbar` isn't specific enough for the detail view toolbar. It won't be used on the list view and I can't think where else it will be needed.

Reply:
Uhh.. soo.. `OutboundShipmentDetailViewToolbar` ? 😁

Reply:
sigh.